src/harpstrata/Pozition/types/index.ts

```diff
@@ -1,18 +1,21 @@
+import type { HalfstepIndex } from '../../Apparatus'
+
 export enum PozitionIds {
   First = 'FIRST',
-  Eighth = 'EIGHTH',
+  Second = 'SECOND',
   Third = 'THIRD',
-  Tenth = 'TENTH',
+  Fourth = 'FOURTH',
   Fifth = 'FIFTH',
-  Twelfth = 'TWELFTH',
+  Sixth = 'SIXTH',
   Seventh = 'SEVENTH',
-  Second = 'SECOND',
+  Eighth = 'EIGHTH',
   Ninth = 'NINTH',
-  Fourth = 'FOURTH',
+  Tenth = 'TENTH',
   Eleventh = 'ELEVENTH',
-  Sixth = 'SIXTH',
+  Twelfth = 'TWELFTH',
 }
 
 export type Pozition = {
   readonly id: PozitionIds;
+  readonly rootOffset: HalfstepIndex;
```

Comment:
Perhaps a rootOffset should be a dedicated type. This would then allow us to have an alternate POZITION_INSTANCES map which could be keyed on the rootOffset. The reason this is required is because we would only want to have 12 entires in that map, just like the map by id. If we use HalfsepIndex then we would need more than 30 entires otherwise we would have to deal with possible undefined returns.